### DISCUSSION TITLE
Name of open pose of the video

### DISCUSSION THREAD

**Discussion by Golgoth23fr** [toyxyz/ComfyUI_toyxyz_test_nodes#56] [Q&A] [answered]
Thanks for those awesome nodes, i use them since a while without any problem :)

But what is the openpose editor used in the video ?

<img width="203" alt="Capture" src="https://github.com/user-attachments/assets/22d200d4-ea7c-4630-b0e8-88b3eb8b9353" />

**Comment by toyxyz** [comment, marked as answer]
https://toyxyz.gumroad.com/l/ciojz

**Reply by Golgoth23fr** [reply]
Thank you !
